### PR TITLE
Add k8s.io namespaces and gsuite group

### PR DESCRIPTION
**Pull request by ThomasWo**
Adds:
- k8s.io prod and canary namespace yamls
- k8s-infra-k8s-io-admins group

Part of #504

### CODE REVIEW

**Comment by ThomasWo** [comment]
@k8s-ci-robot I've signed the CLA

File: k8s.io/namespace-canary.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (thockin) [inline k8s.io/namespace-canary.yaml]: As nted - we should not need either of these files.

**Comment by ThomasWo** [comment]
@thockin I've addressed your feedback (moving the namespace changes into `ensure-namespaces.sh`), PTAL

**Comment by thockin** [comment]
Activated

**Pull request merged**